services/app/src/components/webFooter/webFooter.css

```diff
@@ -0,0 +1,26 @@
+
+#web-footer-container {
+    background-color: #373F41;
+    width: 100%;
+    height: 5%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    color: #ffffff;
+    font-size: 1rem;
+}
+
+#footer-p {
+    letter-spacing: .1px;
+    padding-left: 2rem;
+}
+
+#zenitech-link {
```

Comment:
You're using `id`s extensively. I'd suggest to lean more toward classes as ids are meant for elements that are unique in the application, it implies that they are not meant to be reused. Styles like these should be generic (e.g. `.link`).
Also ids cannot be combined together, classes can, please replace the ids with classNames.